clusterloader2/pkg/util/cluster.go

```diff
@@ -208,3 +209,12 @@ func LegacyIsMasterNode(node *corev1.Node) bool {
 	}
 	return false
 }
+
+func IsControlPlaneNode(node *corev1.Node) bool {
```

Comment:
I am not sure if a new function is needed here. The comment in `LegacyIsMasterNode()` should be clarified first:
https://github.com/kubernetes/perf-tests/blob/daebee8973c392f74b98329574cf9d6a8d5ea246/clusterloader2/pkg/util/cluster.go#L188-L189

Reply:
Yeah this is the reason I didn't update the `LegacyIsMasterNode` function, I am open to other suggestions. By default, kops control plane nodes dont have this label `node.kubernetes.io/exclude-disruption`. It has the following labels-
```
labels:
    beta.kubernetes.io/arch: arm64
    beta.kubernetes.io/instance-type: c6g.xlarge
    beta.kubernetes.io/os: linux
    failure-domain.beta.kubernetes.io/region: us-west-2
    failure-domain.beta.kubernetes.io/zone: us-west-2a
    kops.k8s.io/kops-controller-pki: ""
    kubernetes.io/arch: arm64
    kubernetes.io/hostname: i-00c0f26af0f1a1e81
    kubernetes.io/os: linux
    node-role.kubernetes.io/control-plane: ""
    node.kubernetes.io/exclude-from-external-load-balancers: ""
    node.kubernetes.io/instance-type: c6g.xlarge
    topology.ebs.csi.aws.com/zone: us-west-2a
    topology.kubernetes.io/region: us-west-2
    topology.kubernetes.io/zone: us-west-2a
```
Based on this comment, not sure if `node.kubernetes.io/exclude-disruption` is the right way to detect the control plane instance.

https://github.com/kubernetes/kubernetes/blob/d9714078f821ccae1e2d50ea69c081d73a255f1f/pkg/controller/nodelifecycle/node_lifecycle_controller.go#L808-L810